application-admintools-default/src/main/java/com/xwiki/admintools/internal/ConfigurationManager.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.admintools.internal;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+
+import com.xwiki.admintools.internal.util.ConfigurationInfoUtil;
+
+/**
+ * Manage existing instances.
+ *
+ * @version $Id$
+ * @since 1.0
+ */
+@Component(roles = ConfigurationManager.class)
+@Singleton
+public class ConfigurationManager
+{
+    /**
+     * Contains useful functions for retrieving configuration data.
+     */
+    @Inject
+    private final ConfigurationInfoUtil configurationInfoUtil = new ConfigurationInfoUtil();
```

Comment:
That's not how a component is supposed to be inject (the component manager is in charge of create the instance of ConfigurationInfoUtil, so it does not really make much sense to set anything here).